templates/server.conf.j2

```diff
@@ -57,3 +59,9 @@ remote-cert-tls client
 management /var/run/openvpn/management unix
 management-client-user root
 {% endif %}
+
+{% if openvpn_use_ldap %}
+### LDAP AUTH ###
+plugin /usr/lib64/openvpn/plugin/lib/openvpn-auth-ldap.so "/etc/openvpn/auth/ldap.conf"
```

Comment:
You are right,  I haven't tested on 32 bit. It should be lib instead of lib64. 
The plugin is only installed if the boolean [concerning LDAP is True](https://github.com/kyl191/ansible-role-openvpn/pull/14/files#diff-468007bd6f59ca63e12c1a3613323a17R15).